pipeline-examples/parallel-multiple-nodes/ParallelMultipleNodes.groovy

```diff
@@ -0,0 +1,12 @@
+def labels = ['precise', 'trusty'] // labels for Jenkins node types we will build on
+def builders = [:]
+for (x in labels) {
+    def label = x
+    builders[label] = {
```

Comment:
I suggest adding a comment here that says something to the effect of "building a map to pass into the `parallel` step so we can fire off all the builds at once`